### PR TITLE
Add a basic CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   and make sure it is in your path.
+ # Create a `.gclient` file in an empty directory with the following contents:
+
+```
+solutions = [
+  {
+    "managed": False,
+    "name": "src",
+    "url": "git@github.com:<your_name_here>/sky_engine.git",
+    "custom_deps": {},
```
Comment (eseidelGoogle) [inline CONTRIBUTING.md]: Is this needed?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+ # Fork https://github.com/domokit/sky_engine into your own GitHub account.
+ # [Download depot_tools](http://www.chromium.org/developers/how-tos/install-depot-tools)
+   and make sure it is in your path.
+ # Create a `.gclient` file in an empty directory with the following contents:
+
+```
+solutions = [
+  {
+    "managed": False,
```
Comment (eseidelGoogle) [inline CONTRIBUTING.md]: I'm surprised this is even still an option.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "name": "src",
+    "url": "git@github.com:<your_name_here>/sky_engine.git",
+    "custom_deps": {},
+    "deps_file": "DEPS",
+    "safesync_url": "",
```
Comment (eseidelGoogle) [inline CONTRIBUTING.md]: We can also clean this up later.

**Comment by eseidelGoogle** [comment]
lgtm

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ 4. `svn ls https://dart.googlecode.com/` and accept the certificate permanently.
+ 5. `gclient sync`
+ 6. `cd src`
+ 7. `git remote add upstream git@github.com:domokit/sky_engine.git`
+ 8. `./build/install-build-deps.sh`
+
+Building the code
+-----------------
+
+ 1. `./mojo/tools/mojob gn`
```
Comment (collinjackson) [inline CONTRIBUTING.md]: should be mojob.py

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  },
+]
+target_os = ["android"]
+```
+
+ 4. `svn ls https://dart.googlecode.com/` and accept the certificate permanently.
+ 5. `gclient sync`
+ 6. `cd src`
+ 7. `git remote add upstream git@github.com:domokit/sky_engine.git`
+ 8. `./build/install-build-deps.sh`
```
Comment (collinjackson) [inline CONTRIBUTING.md]: This last step errors out on Mac.

**Pull request merged**